dotcom-rendering/src/amp/components/topMeta/TopMetaLiveblog.tsx

```diff
@@ -86,7 +86,6 @@ const Headline: React.FC<{
 	headlineText: string;
 	standfirst: string;
 	pillar: Theme;
-	starRating?: number;
```

Comment:
What's the reason that star rating has been removed as part of this work?

Reply:
This prop was not used anywhere within the `topMetaLiveblog` component, after updating, ESLint identified it and threw an error as unused prop.

Reply:
(Despite it being passed in by a parent component, it isn't used within the component, making it redundant)

Reply:
Ah, got it.